steal-stache.js

```diff
@@ -5,21 +5,30 @@ var loader = require("@loader");
 var addImportSpecifiers = require("steal-config-utils/import-specifiers").addImportSpecifiers;
 
 function template(imports, intermediate, filename){
+	var tagImportNames = JSON.stringify(imports.slice(6));
 	imports = JSON.stringify(imports);
 	intermediate = JSON.stringify(intermediate);
 
-	return "define("+imports+",function(module, assign, stache, mustacheCore){ \n" +
+	return "define("+imports+",function(module, assign, stache, mustacheCore, canViewImport, canStacheBindings){ \n" +
 		(filename ?
 			"\tvar renderer = stache(" + JSON.stringify(filename) + ", " + intermediate + ");\n" :
 			"\tvar renderer = stache(" + intermediate + ");\n"
 		) +
+		"\tvar tagImports = Array.prototype.slice.call(arguments, 6);\n" +
 		"\treturn function(scope, options, nodeList){\n" +
 		"\t\tvar moduleOptions = assign({}, options);\n" +
+		"\t\tvar tagImportMap = " + tagImportNames + ".reduce((map, name, index) => {\n" +
+		"\t\t\tmap[name] = tagImports[index];\n" +
+		"\t\t\treturn map;\n" +
+		"\t\t}, {});\n" +
+		"\n"+
 		"\t\tif(moduleOptions.helpers) {\n" +
-		"\t\t\tmoduleOptions.helpers = assign({ module: module }, moduleOptions.helpers);\n" +
+		"\t\t\tmoduleOptions.helpers = assign({ module: module, tagImportMap: tagImportMap }, moduleOptions.helpers);\n" +
 		"\t\t} else {\n" +
 		"\t\t\tmoduleOptions.module = module;\n" +
+		"\t\t\tmoduleOptions.tagImportMap = tagImportMap;\n" +
```

Comment:
this may be unnecessary? wasn't clear to me in what cases the `helper` object would be missing

Reply:
yeah, why is this being done?

Reply:
removed this in favor of always passing `module` as part of the LetContext. If someone knows why it needed to be done this way we can add it back.